sprocket-doc.md

```diff
@@ -0,0 +1,63 @@
+# How to make the most of Sprocket doc
+
+Sprocket is capable of rendering rich HTML documentation for any WDL workspace! However to make the most of this tool, there are certain documentation conventions which should be followed. Of course you are free to ignore this document, and Sprocket should do a passable job of documenting your WDL, but there may be some awkward rendering or unexpected behavior if you aren't conforming to what the tool expects.
+
+## General structure
+
+The generated documentation directory (named `docs` by default) is completely self contained and can be moved, zipped, and shared without any of the raw WDL files it documents. Please let us know if you run into any issues while sharing your documentation!
+
+The `docs` directory will always contain a `style.css` file, an `index.js` file, an `index.html` file, and an `assets/` directory filled with SVG icons and logos. The rest of the contents are dynamically created based on the structure of the WDL workspace being documented. Each WDL file found in the workspace will be represented as a directory within `docs`. The relative path from the root of the workspace to each WDL file is copied in the `docs` directory to keep your documentation organized.
+
+The directory for each WDL file will have an `index.html` file and one HTML file per struct, task, or workflow defined in the WDL.
+
+## Custom themes
```

Comment:
I'm not sure we need this. I think it gets left undocumented. Maybe a short blurb along the lines of "We provide a single styling that should be widely useful. If you want to customize the styling, please let us know.".